runner/execer/fake/intercept.go

```diff
@@ -0,0 +1,31 @@
+package fake
```

Comment:
is this really a fake, my understanding is this was intending to be ran side by side with OS Execer in Prod.  Fake implies test use only to me.